docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+### How will the Polkadot Relay Chain connect to external chains in the ecosystem?
 
-### What are the ways to find out the minimum stake necessary for the validators?
+One of the cornerstone interoperability technologies being research and developed
+for deployment on Polkadot is cross-chain bridges. Bridges come in a variety
+of flavors with varying levels of trust associated with them. Polkadot is predominantly
+researching the trust-minimized flavor that imposes economic costs on the operators
+of the bridge, and therefore makes it economically secure. Bridge efforts are being
+work on in concert with other projects in the ecosystem. Eventually, there
+will be bridges between Polkadot and most of the other major chains. 
 
-There are a few ways to estimate it.
+## DOTs
 
-- [Offline Phragmen](https://github.com/kianenigma/offline-phragmen)
+### What is the inflation rate of the DOT?
 
-If you want to know what would be the outcome of the election in the next era, this can provide an estimation result. You can customize parameters such as number of validators to elect, network, and WebSocket endpoint, but it would take some time to build the binary.
-- [Validator stats script](https://github.com/ansonla3/kusama-validator-stats)
+The inflation rate is 10% per year.
 
-This script helps you quickly identify which validator is the lowest-staked and the basic statistics about staking on Kusama.
+A portion of the inflation is rewarded to validators for performing their duties,
+while another portion may go directly to the treasury. The exact percentage that
+goes into both varies and is based on the amount of DOTs that are staked.
+Please see the article on [inflation](learn-staking#inflation) for more information.
 
-- [Copy everything from the Staking page](https://polkadot.js.org/apps/#/staking) 
+### Can I buy or transfer DOT tokens?
 
-It is also possible to copy both `own stake` and `other stake` on the Staking page to a spreadsheet and then do a simple calculation.
+Testnet DOT tokens are freely available from a variety of sources. See
+the [DOT page](learn-DOT) for details.
+
+The Web3 Foundation will distribute up to 20% of mainnet DOTs prior to network
+launch (see [Light Paper](https://polkadot.network/Polkadot-lightpaper.pdf) or
+the [Polkadot Network FAQ](https://polkadot.network/faq/)).  Subscribe to the
+Polkadot newsletter on [polkadot.network](https://polkadot.network/) for further
+updates.
+
+DOT token are not transferrable until the launch of Polkadot Beta is complete.
+Any transfers before that time of DOTs are illegitimate and unauthorized. DOTs
+are currenly represented on Ethereum as the DOT Indicator Token, these cannot be
```

Comment:
```suggestion
are currengly represented on Ethereum as the DOT Indicator Token, these cannot be
```

Reply:
Should we include a link to the Dot Indicator token contract?

Reply:
```suggestion
are currently represented on Ethereum as the DOT Indicator Token, these cannot be
```

Reply:
@laboon good idea